docs_rtd/_templates/layout.html

```diff
@@ -168,193 +168,80 @@
-                  fill="currentColor"
-                >
-                  <g
-                    id="discuss"
-                    stroke="none"
-                    stroke-width="1"
-                    fill="none"
-                    fill-rule="evenodd"
-                  >
-                    <g id="discuss" transform="translate(19.000000, 0.000000)">
-                      <path
-                        d="M171.371338,721.5 L-9.5,910.604241 L-9.5,222 C-9.5,167.047667 35.0476674,122.5 90,122.5 L788,122.5 C842.952333,122.5 887.5,167.047667 887.5,222 L887.5,622 C887.5,676.952333 842.952333,721.5 788,721.5 L171.371338,721.5 Z"
-                        id="Rectangle-Copy"
-                        stroke="#048AFC"
-                        stroke-width="19"
-                        fill="#048AFC"
-                      ></path>
-                      <path
-                        d="M82.5,787.956652 L236.238837,617.116934 L875,617.116934 C918.354353,617.116934 953.5,581.971287 953.5,538.616934 L953.5,90.9276217 C953.5,47.5732688 918.354353,12.4276217 875,12.4276217 L161,12.4276217 C117.645647,12.4276217 82.5,47.5732688 82.5,90.9276217 L82.5,787.956652 Z"
-                        id="Rectangle"
-                        stroke="#048AFC"
-                        stroke-width="23"
-                        fill="#FFFFFF"
-                      ></path>
-                      <circle
-                        id="Oval"
-                        fill="#048AFC"
-                        cx="525"
-                        cy="306"
-                        r="65"
-                      ></circle>
-                      <circle
-                        id="Oval-Copy"
-                        fill="#048AFC"
-                        cx="714"
-                        cy="306"
-                        r="65"
-                      ></circle>
-                      <circle
-                        id="Oval-Copy-2"
-                        fill="#048AFC"
-                        cx="345"
-                        cy="306"
-                        r="65"
-                      ></circle>
-                    </g>
-                  </g>
-                </svg>
-              </i>
-              Discuss
-            </a>
+            <a class="header-item" href="https://greatexpectations.io/expectations"
+              >EXPECTATIONS GALLERY</a
+            >
+          </li>
+          <li>
+            <a class="header-item" href="https://greatexpectations.io/blog"
+              >BLOG</a
+            >
+          </li>
+          <li>
+            <a class="header-item" href="https://greatexpectations.io/case-studies"
+              >CASE STUDIES</a
+            >
           </li>
+          <li>
+            <a class="header-item" href="https://greatexpectations.io/case-studies"
+              >ABOUT US</a
+            >
+          </li>
+          <li>
+            <a class="header-item" href="https://greatexpectations.io/case-studies"
+              >CAREERS</a
```

Comment:
Should these go to different pages (i.e. not the case-studies page)? Or are those pages not yet ready?